site/content/en/docs/installation/_index.md

```diff
@@ -108,7 +108,8 @@ the default Kueue Configuration
 struct ([v1beta1@main](https://pkg.go.dev/sigs.k8s.io/kueue@main/apis/config/v1beta1#Configuration)).
 The contents of the ConfigMap are similar to the following:
 
-> __The `namespace` and `internalCertManagement` fields are available in Kueue v0.3.0 and later__
+> __The `namespace`, `waitForPodsReady`, and `internalCertManagement` fields are available in Kueue v0.3.0 and later.
```

Comment:
yeah, this can go away

Reply:
Removed the v0.3.0 field caveats; left just 1 copy of the field added in 0.7.0 caveat